src/test/java/me/devyonghee/racingcar/model/sample/RandomEngineSample.java

```diff
@@ -0,0 +1,10 @@
+package me.devyonghee.racingcar.model.sample;
+
+import me.devyonghee.racingcar.model.RandomEngine;
+
+import java.util.Random;
+
+public class RandomEngineSample {
+
+    public static final RandomEngine TEM_LIMIT_RANDOM_ENGINE = RandomEngine.from(new Random(), 1);
```

Comment:
샘플 상수 필드를 잘 구현해주셨네요. 👍 
10인것 같은데 맞나요?
```suggestion
    public static final RandomEngine TEM_LIMIT_RANDOM_ENGINE = RandomEngine.from(new Random(), 10);
```
구현하신다면 `RandomEngine`의 테스트 케이스들도 구현해보시면 좋을 것 같아요. 😄

Reply:
아.. 0이 빠졌군요.. 바로 수정하겠습니다.
`RandomEngine` 에 대한 테스트는 `RandomEngineTest` 이곳에 정의했고 sample 만 별도의 클래스로 분리했습니다!!